src/persist/src/lib.rs

```diff
@@ -45,8 +40,6 @@ use crate::persister::Persister;
 //   them as type parameters? Materialize will only be using one combination of
 //   them (two with `()` vals?) but the generality might make things easier to
 //   read. Of course, it also might make things harder to read.
-// - Is PersistManager getting us anything over a type alias? At the moment, the
```

Comment:
At this exact moment, I think this is still useful, but my in-progress PR to move work into a separate thread makes Persister clone-able and threadsafe, so it's going away anyway.